File: ds_&_algorithms/algorithm_analysis/linear_time_complexity/go/LinTimeComp.go
package main

import "fmt"

func sumOperation(n int) int {
	var sum int = 0

	for i := 1; i <= n; i++ {
		sum += i
	}
	return sum
}

func main() {
	var value int = 3
	var finalSum = sumOperation(value)

	fmt.Printf("The value of the operation is: %v", finalSum)
}

File: ds_&_algorithms/algorithm_analysis/linear_time_complexity/java/LinTimeComp.java
package algorithm_analysis.linear_time_complexity.java;

public class LinTimeComp {

    static int sumOperation(int n){
        int sum = 0;
        for (int i = 1; i<=n; i++){
            sum = sum + i; // 
        }
        return sum;
    }

    public static void main(String[] args){
        int val = 3;
        int finalSum = sumOperation(val);

        System.out.println("The value of the sum is:" +finalSum);
    }
    
}
